openedx/features/survey_report/admin.py

```diff
@@ -0,0 +1,52 @@
+"""
+Django Admin page for SurveyReport.
+"""
+
+
+from django.contrib import admin
+from .models import SurveyReport
+
+
+class SurveyReportAdmin(admin.ModelAdmin):
+    """
+    Admin to manage survey reports.
+    """
+    readonly_fields = (
+        'courses_offered', 'learners', 'registered_learners',
+        'enrollments', 'generated_certificates', 'extra_data',
+        'created_at'
+    )
+
+    list_display = (
+        'id', 'summary', 'created_at'
+    )
+
+    def summary(self, obj) -> str:
+        """
+        Show a summary of the survey report.
+        info:
+        - Courses: Total number of active unique courses.
+        - Learners: Recently active users with login in some weeks.
+        - Enrollments: Total number of active enrollments in the platform.
+        """
+        return f"Courses: {obj.courses_offered}, Learners: {obj.learners}, Enrollments: {obj.enrollments}"
```

Comment:
I think we should always specify "active learners" or "total learners". Leaving the "learners" without the qualifier could always be a source of confusion